app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,35 @@
+
+require_relative '../../lib/TunesTakeoutWrapper'
+
+
+class SuggestionsController < ApplicationController
+  attr_reader :show
+
```

Comment:
I'm not sure why you have an attr_reader for this... Are you accessing this method outside this class somewhere?